packages/nextcloud/lib/src/webdav/file.dart

```diff
@@ -28,7 +27,7 @@ class WebDavFile {
     return PathUri(
       isAbsolute: false,
       isDirectory: href.isDirectory,
-      pathSegments: href.pathSegments.sublist(webdavBase.pathSegments.length),
+      pathSegments: href.pathSegments.sublist(PathUri.webdav().pathSegments.length),
```

Comment:
Shouldn't this still use the base?

Reply:
I was looking into that too, it is not ideal, however getting the actual base path there is not trivial due to the XML handling. I will be thinking about this one more time but wanted general feedback first.